### ISSUE TITLE
[Bug] Crash when click on Notifications in ComposeActivity details

### ISSUE THREAD

**Issue by rezazarchi** [moezbhatti/qksms#1642]
### DESCRIPTION
Crash happens when click on Notifications in ComposeActivity details and device language is Farsi (Persian)

### STEPS
1. set system language to Persian
2. Open Application
3. Open a conversation
4. click on more...
5. click on notifications (اعلانها)

### EXPECTED
Should open notifications settings

### OBSERVATIONS
crash happens. logcat:
```
2020-09-10 13:27:54.848 5416-5416/com.moez.QKSMS E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.moez.QKSMS, PID: 5416
    io.reactivex.exceptions.OnErrorNotImplementedException: length=5; index=5
    Caused by: java.lang.ArrayIndexOutOfBoundsException: length=5; index=5
        at com.moez.QKSMS.common.base.QkViewModel.<init>(QkViewModel.kt:45)
        at [[ ↑↑ Inferred subscribe point ↑↑ ]].(:0)
        at [[ ↓↓ Original trace ↓↓ ]].(:0)
        at com.moez.QKSMS.feature.notificationprefs.NotificationPrefsViewModel$6$1.invoke(NotificationPrefsViewModel.kt:71)
        at com.moez.QKSMS.feature.notificationprefs.NotificationPrefsViewModel$6$1.invoke(NotificationPrefsViewModel.kt:39)
        at com.moez.QKSMS.common.base.QkViewModel$1.apply(QkViewModel.kt:44)
        at com.moez.QKSMS.common.base.QkViewModel$1.apply(QkViewModel.kt:32)
        at io.reactivex.internal.operators.observable.ObservableScanSeed$ScanSeedObserver.onNext(ObservableScanSeed.java:98)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:201)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6077)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)


```

### solution
This crash happens because the outOfBound of string array in Farsi Strings resource. I'll fix this issue

**Comment by rezazarchi** [comment]
fixed in #1643 1643

**Comment by moezbhatti** [comment]
Thanks for the report. This is actually not an issue in the current release - and it'll be fixed once I import the latest translations from Crowdin before the next release.
